app/code/Magento/Bundle/Model/Product/Type.php

```diff
@@ -285,7 +290,12 @@ public function getRelationInfo()
      */
     public function getChildrenIds($parentId, $required = true)
     {
-        return $this->_bundleSelection->getChildrenIds($parentId, $required);
+        $cacheKey = $parentId . '-' . ($required ? 1 : 0);
```

Comment:
AFAIK $parentId here can be array here.

Reply:
No, it isn't.  In the annotation to the function, the specified parameter takes only "int".

>    *
     * @param int $parentId
     * @param bool $required
     * @return array
     */
    public function getChildrenIds($parentId, $required = true)
    {
        $cacheKey = $parentId . '-' . ($required ? 1 : 0);